eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -402,34 +409,35 @@ object Helpers {
      */
     def makeFirstCommitTxs(keyManager: ChannelKeyManager, channelConfig: ChannelConfig, channelFeatures: ChannelFeatures, temporaryChannelId: ByteVector32,
                            localParams: LocalParams, remoteParams: RemoteParams,
-                           localFundingAmount: Satoshi, remoteFundingAmount: Satoshi, pushMsat: MilliSatoshi,
+                           localFundingAmount: Satoshi, remoteFundingAmount: Satoshi,
+                           localPushAmount: MilliSatoshi, remotePushAmount: MilliSatoshi,
                            commitTxFeerate: FeeratePerKw,
                            fundingTxHash: ByteVector32, fundingTxOutputIndex: Int,
                            remoteFirstPerCommitmentPoint: PublicKey): Either[ChannelException, (CommitmentSpec, CommitTx, CommitmentSpec, CommitTx)] = {
-      val toLocalMsat = if (localParams.isInitiator) localFundingAmount.toMilliSatoshi - pushMsat else localFundingAmount.toMilliSatoshi + pushMsat
-      val toRemoteMsat = if (localParams.isInitiator) remoteFundingAmount.toMilliSatoshi + pushMsat else remoteFundingAmount.toMilliSatoshi - pushMsat
+      val fundingAmount = localFundingAmount + remoteFundingAmount
+      val toLocalMsat = localFundingAmount.toMilliSatoshi - localPushAmount + remotePushAmount
+      val toRemoteMsat = remoteFundingAmount.toMilliSatoshi + localPushAmount - remotePushAmount
```

Comment:
So, this is the only part where we actually use the _push_ amount. I was expecting something more complex with impacts in `InteractiveTxBuilder` but I was wrong, we only care about the contributions when building the funding tx, whereas _push_ amounts affect the first commitment tx. Cool!

Reply:
> we only care about the contributions when building the funding tx, whereas push amounts affect the first commitment tx

That's exactly it, that is also why I chose to leave push amounts out of `InteractiveTxParams`.